job-server/src/main/resources/shiro.ini

```diff
@@ -0,0 +1,12 @@
+
+# use this for basic ldap authorization, without group checking
+# activeDirectoryRealm = org.apache.shiro.realm.ldap.JndiLdapRealm
+# use this for checking group membership of users based on the 'member' attribute of the groups:
+activeDirectoryRealm = spark.jobserver.auth.LdapGroupRealm
+activeDirectoryRealm.contextFactory.environment[java.naming.security.credentials] = password
+activeDirectoryRealm.contextFactory.url = ldap://xxx.yyy..org:389

```

Comment:
So this file won't work as is, right?  Instead of bundling it in src/main/resources, I'd prefer to stick a template in the `config/` subdir, just like we have for the other configs like logging, and then make it part of the deploy process + Docker container.